packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -7,25 +7,38 @@ import snakeCase from 'lodash/snakeCase';
 import uniq from 'lodash/uniq';
 
 import Image from './images.class';
-import { IMAGES_REGEX } from './images.constants';
-
-function getDistribution(name, type) {
-  const os = IMAGES_REGEX[type];
-  if (os) {
-    const distribution = os.find((distrib) => distrib.regex.test(name));
-
-    return distribution?.name || ''.concat(type, '_other');
-  }
-
-  return 'unknown';
-}
+import {
+  IMAGES_REGEX,
+  REGEX_PREFIX_BAREMETAL,
+  PREFIX_BAREMETAL_LENGTH,
+} from './images.constants';
 
 export default class ImagesList {
   /* @ngInject */
   constructor($http, OvhApiCloudProjectImage, OvhApiCloudProjectSnapshot) {
     this.$http = $http;
     this.OvhApiCloudProjectImage = OvhApiCloudProjectImage;
     this.OvhApiCloudProjectSnapshot = OvhApiCloudProjectSnapshot;
+    this.REGEX_PREFIX_BAREMETAL = REGEX_PREFIX_BAREMETAL;
+    this.PREFIX_BAREMETAL_LENGTH = PREFIX_BAREMETAL_LENGTH;
+  }
+
+  removeBaremetalPrefix(imageType) {
+    if (imageType.match(this.REGEX_PREFIX_BAREMETAL)) {
+      return imageType.slice(this.PREFIX_BAREMETAL_LENGTH);
```

Comment:
i think you want to use a capturing regex here, something like : /^baremetal-(.*)/ instead of slicing

Reply:
It's fixed